model/config.go

```diff
@@ -416,6 +417,10 @@ func (s *ServiceSettings) SetDefaults(isUpdate bool) {
 		s.EnableDeveloper = NewBool(false)
 	}
 
+	if !*s.EnableDeveloper || s.DeveloperFlags == nil {
+		s.DeveloperFlags = []string{}
```

Comment:
@hmhealey might be able to correct me, but I think we want this to just be a `string` in the configuration vs. having the enduser actually configure this as an array.

Reply:
Not Harrison, but yes, this should be a string.

Reply:
If we're going with a string, is there a preferred format for this string? As this string would still contain a list of values like `unsafe-eval` , `unsafe-inline` etc